README.md

```diff
@@ -222,6 +230,8 @@ See the documentation of `eglot-x-enable-ff-related-file-integration`.
 [View Mir]: https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#view-mir
 [View Recursive Memory Layout]: https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#view-recursive-memory-layout
 [Workspace Symbols Filtering]: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#workspace-symbols-filtering
+[clangd inactive code]: https://clangd.llvm.org/features#kinds
+[ccls skipped ranges]: https://github.com/MaskRay/ccls/blob/f36ecb0c0e025f3f3a5c2d28c823316e5d0c48ba/src/message_handler.cc#L275
```

Comment:
Can you, please, run sort-lines on this region?

(It is strange that this ccls feature is completely undocumented.)